sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/util/Utilities.java

```diff
@@ -148,12 +149,21 @@ public static <T> T[] arraySlice(T[] data, int start) {
     }
 
     public static void runProcess(String directory, String... commands) throws IOException, InterruptedException {
-        ProcessBuilder processBuilder = new ProcessBuilder();
-        processBuilder.command(commands);
-        processBuilder.directory(new File(directory));
-        processBuilder.inheritIO();
+        File out = File.createTempFile("out", ".tmp", new File("."));
+        out.deleteOnExit();
+        ProcessBuilder processBuilder = new ProcessBuilder()
+                .command(commands)
+                // If this is called from a JUNIT test the output
+                // of the process interferes with the surefire plugin communication,
+                // so we need to redirect the output.
+                .directory(new File(directory))
+                .redirectOutput(out)
```

Comment:
stderr and stdout going to the same file?

Reply:
yes, I want them interleaved in the order they are produced if possible.